cmd/format.go

```diff
@@ -1,22 +1,43 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/formatter"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+const failFastFlagName = "fail-fast"
+
 func setupFormatCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "format",
 		Short: "Format the package",
 		Long:  "Use format command to format the package files.",
 		RunE:  formatCommandAction,
 	}
+	cmd.Flags().BoolP(failFastFlagName, "f", false, "fail if any file requires formatting")
```

Comment:
Nit: to reflect the "fast" part of fail fast:

```suggestion
	cmd.Flags().BoolP(failFastFlagName, "f", false, "fail immediately if any file requires formatting")
```

Reply:
Fixed.